__init__.py

```diff
@@ -142,6 +142,16 @@ def execute(self, context):
         return handle_errors(self, shell.create_lods, {})
 
 
+class GenerateRigEdit(bpy.types.Operator):
+    """Generate rig edit nodes for all posed bones"""
+    bl_idname = "pose.generate_rig_edit"
```

Comment:
Maybe more generic? Is there anything else we might add?

Reply:
brain dead moment. I thought this was the panel, not the operator -.-

Reply:
Maybe really call this 'Pose to Rig Edit'?

Reply:
I don't know what else could require automation regarding rigs. Maybe for animations automatically fixing the IK setups. Could be "Generate Rig Edit Nodes"
I was thinking of adding options for
-apply armature modifiers of all armature children
-apply rest pose

Reply:
Yeah, so those would go into the panel too, probably as separate operators?

Reply:
changed name to "Generate Rig Edit from Pose"